.circleci/config.yml

```diff
@@ -5,4 +5,4 @@ jobs:
       - image: hamroctopus/shellcheck:1.0.0
     steps:
       - checkout
-      - run: shellcheck src/*
\ No newline at end of file
+      - run: shellcheck -e SC2059 src/*
```

Comment:
Why add this?

Reply:
Had to add this flag to ignore rule SC2059 which states 
```         
^-- SC2059: Don't use variables in the printf format string. Use printf "..%s.." "$foo".
```

Reply:
Why not do what it says instead? `printf "%s" "$changelog"`

Reply:
I didn't know if formatting would be an issue. If you do `printf "%s" "$changelog"` the output is ```
\n\n\n### Bug Fixes\n\n* **aget:** Enable writing json uri mode ([#331](http://dequelabs/attest-node-suite/issues/331)) ([d3ffc6a](http://dequelabs/attest-node-suite/commits/d3ffc6a))\n* **aget:** stub util test properly to allow for cross platform testing ([#280](http://dequelabs/attest-node-suite/issues/280)) ([8dd851b](http://dequelabs/attest-node-suite/commits/8dd851b))\n* **aget:** Support Node v8 ([#264](http://dequelabs/attest-node-suite/issues/264)) ([da94013](http://dequelabs/attest-node-suite/commits/da94013))\n* **attest-standards:** Allow from a dynamic list of tags ([#295](http://dequelabs/attest-node-suite/issues/295)) ([e885721](http://dequelabs/attest-node-suite/commits/e885721))\n\n\n\n\n```

Reply:
Maybe we should run this script and actually test what's happening...lmk if you want to pair on it.

Reply:
But yes, I think we do want everything escaped like that

Reply:
We can pair on it just msg me when.